extensions/unsplash/src/components/Actions.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ActionPanel,
+  Icon,
+  useNavigation,
+  getPreferenceValues,
+  OpenInBrowserAction,
+  CopyToClipboardAction,
+} from "@raycast/api";
+
+// Functions
+import { saveImage } from "@/functions/saveImage";
+import { setWallpaper } from "@/functions/setWallpaper";
+import { copyFileToClipboard } from "@/functions/copyFileToClipboard";
+
+// Components
+import Details from "@/views/Details";
+
+// Types
+interface BaseProps {
+  item: SearchResult;
+  back?: boolean;
+  details?: boolean;
+}
+
+export const Actions: React.FC<BaseProps> = ({ back = false, details = false, item }) => (
+  <ActionPanel>
+    <Sections back={back} details={details} item={item} />
+  </ActionPanel>
+);
+
+export const Sections: React.FC<BaseProps> = ({ back = false, details = false, item }) => {
+  const { pop, push } = useNavigation();
+  const { downloadSize } = getPreferenceValues<UnsplashPreferences>();
+
+  const imageUrl = item.urls?.raw || item.urls?.full || item.urls?.regular || item.urls?.small;
+
+  const clipboardCopyUrl = {
+    url: item.urls?.[downloadSize] || imageUrl,
+    id: `${item.id}-${downloadSize}`,
+  };
+
+  return (
+    <>
+      <ActionPanel.Section>
+        {details && (
+          <ActionPanel.Item title="Show Details" icon={Icon.List} onAction={() => push(<Details result={item} />)} />
+        )}
+
+        {back && <ActionPanel.Item icon={Icon.Window} title="Go Back" onAction={pop} />}
```

Comment:
You don't need that. Just press ESC ;-)

Reply:
Should I? Because sometimes I click on the wrong image or I decide the image isn't the one I want so I just click Enter as it is the option at the top and goes directly to back (it'll open the browser and Unsplash website if I remove this, and that might be annoying if your browser wasn't running at that time). Enter is also easier to reach than the Escape key. Though I leave it to your decision, this is more of a personal POV and I don't know if more people would agree with you or me, let me know! 😅

Reply:
It's just to keep the Raycast experience consistent. Folks are used to Escape going back. So no need to introduce a new pattern for this. This would make your extension than inconsistent with the others. I would encourage you to remove it.

Reply:
Alright! That makes sense 🥰